CODEOWNERS

```
@@ -2,6 +2,6 @@
 # the repo. Unless a later match takes precedence,
 # they will be requested for review when someone opens a 
 # pull request.
-*       @kimdhamilton
+*       @sbesson @joshmoore
```

Since this is a feature of codeowners https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners, I'd be tempted to use the `@ome/ngff` team here rather than maintaining a list of individuals. 

As this can get noisy, this would probably require to review the notifications/assignment settings https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-code-review-settings.

---

yeah, could be a team:
```suggestion
*       @ome/ngff
```
but then more of a question -- why bother with CODEOWNERS? ;)